### ISSUE TITLE
How to use plotdiffBaits to show the results of 4 condition

### ISSUE THREAD

**Issue by FN2023** [RegulatoryGenomicsGroup/chicdiff#17]
I ran chicdiffPipeline with 4 condition, where each condition had 2 replicates. In the results directory it creates diffBaitPlot.png, where it shows only 4 samples (2 condition, 2 replicate each) instead of 8 samples (4 condition, 2 replicate each).  

I then used "plotDiffBaits" . and again i see first two conditions. How can i see all 4 conditions? chicdiffPipeline  result files "countput.Rds" shows 4 conditions -mESCWT mESCKO mEpiLCWT mEpiLCKO

baits = c(456463)
plotDiffBaits(outputRds, countputRds, bmapRds, baits = baits)

**Comment by mspivakov** [comment]
To be honest, I'm not sure you can... This function was indeed designed for
pairwise comparisons.

**Comment by FN2023** [comment]
Does the chicdiffPipeline function also runs on two condition only. Meaning the output files of this function only identify diff loops  that were identified in first two conditions.

if true, then i have to run chicdiffPipeline twice. First with WT vs. KO in mESC and then WT vs. KO in Epiblast cells.

**Comment by mspivakov** [comment]
yes, it is currently developed for two conditions. In principle however it
can be modified to take advantage of deseq's multiway analysis. however we
didn't yet get round to that

**Comment by FN2023** [comment]
So, if only two conditions can be compared , then my question is 

Can i prepare one peakMatrix using all 8 samples and then in "setChicdiffExperiment" i use chicagoData and countData based on two tested conditions but peakMatrix file stays the same (having all 8 samples). 

Or do you suggest to make peaks file for each pairwise comparison seperately and then use that in  "setChicdiffExperiment"?

**Comment by mspivakov** [comment]
Yes - you can use features defined in any way you like as the peak matrix.
